src/components/remote_control/src/remote_control_plugin.cc

```diff
@@ -321,9 +321,17 @@ ResourceAllocationManager& RemoteControlPlugin::resource_allocation_manager() {
 }
 
 void RemoteControlPlugin::OnApplicationEvent(
-    functional_modules::ApplicationEvent event, const uint32_t application_id) {
+    functional_modules::ApplicationEvent event,
+    application_manager::ApplicationSharedPtr application) {
   LOG4CXX_AUTO_TRACE(logger_);
-  resource_allocation_manager_.OnApplicationEvent(event, application_id);
+    if (false == service()->IsRemoteControlApplication(application)) {
+      LOG4CXX_DEBUG(logger_,
+                    "Application " << application->app_id()
+                                   << " has no remote control functionality."
+                                   << " Event will be ignored for RC plugin");
+      return;
+    }
```

Comment:
Style issues, run `. tools/infrastructure/check_style.sh --fix` in source folder

Reply:
@jacobkeeler yes, sorry. Style issues was fixed in https://github.com/smartdevicelink/sdl_core/pull/1760/commits/86974f42574b6a1427192bc2842fbbdf82425841